src/exe/Main.hs

```diff
@@ -125,7 +134,21 @@ optionDescriptors =
     (  "Optional. Path to output directory.\n"
     ++ "Prints to the console by default."
     )
+  , Option
+    ['f']
+    ["frontend"]
+    (ReqArg (\f opts -> opts { optFrontend = parseFrontend f }) "FRONTEND")
+    (  "Optional. Frontend to use for input parsing.\n"
+    ++ "Uses haskell-src-extensions by default"
```

Comment:
The list of available front ends should be printed out here.

Reply:
See also <https://github.com/FreeProving/free-compiler/pull/126#discussion_r452441672>.